docs/getting_started.md

```diff
@@ -12,6 +12,7 @@
    Latest client can be downloaded from [oc-client](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/).
 4. For vSphere based installations, [terraform](https://learn.hashicorp.com/terraform/getting-started/install.html)
    and [jq]( https://stedolan.github.io/jq/download/) should be installed ( terraform version should be 0.11.13  )
+5. To install openshift-logging on the cluster [follow](docs/deploy_logging.md)
```

Comment:
you have to go relatively:  ../deploy_logging.md , or absolute path /docs/deploy_logging.md , you can see here:  https://github.com/red-hat-storage/ocs-ci/blob/2ec3dd070f276db9f0e61b737630bf7c8ca88077/docs/getting_started.md  that it doesn't work and returns 404